style.css

```diff
@@ -0,0 +1,54 @@
+body{
+    margin: 0;
+    padding: 0;
+    background: #262626;
```

Comment:
Do you know what is CSS reset ?

Reply:
nope, can you tell?

Reply:
### Different browsers have different default styles, so using ***CSS reset***, you can reset all those default styles and start styling  your project from scratch.
***for learning more you can checkout this*** : https://stackoverflow.com/questions/11578819/css-reset-what-exactly-does-it-do
https://www.webfx.com/blog/web-design/the-history-of-css-resets/